content/en/docs/collector/about.md

```diff
@@ -102,8 +102,8 @@ Prometheus metrics.
 ```bash
 $ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
     cd opentelemetry-collector; make otelcol; \
-    go build examples/main.go; ./main & pid1="$!"; \
-    ./bin/$($GOOS)/otelcol --config ./examples/otel-local-config.yaml; kill $pid1
+    go build examples/demo/app/main.go; ./main & pid1="$!"; \
+    ./bin/$($GOOS)/otelcol --config ./examples/local/otel-config.yaml; kill $pid1
```

Comment:
On my Mac, the binary path is `./bin/otelcol_darwin_amd64`. That is: `./bin/otelcol_$(go env GOOS)_$(go env GOARCH)`. Should we update this binary path.

Reply:
Yeah

Reply:
Cool, done!